code/game/objects/items/defib.dm

```diff
@@ -607,6 +607,8 @@
 					H.Jitter(100)
 					if(tplus > tloss)
 						H.adjustBrainLoss( max(0, min(99, ((tlimit - tplus) / tlimit * 100))), 150)
+					if(defib.healdisk)
+						H.heal_overall_damage(25, 25)
```

Comment:
Why would a defib HEAL Burn and brute damage ??????

Reply:
They already do heal, just make it more.

Reply:
Also do to defibs harming people with o2 damg, it would replace the healed tissue with o2 lost